Hotel.rb

```diff
@@ -0,0 +1,22 @@
+require 'csv'
```

Comment:
Why did this file make it to the submission? It doesn't seem to be doing anything.